### DISCUSSION TITLE
Termination windows "DAYS" period type expansion

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#242] [General]
We might not always be talking about normal days, sometimes it's only business days, or "trading" days. These are tricky though because different jurisdictions have different holidays, different stock exchanges have different days where they're closed.

**Comment by JSv4** [comment]
This is a good point, but it's probably not a quick fix. We kicked around the idea of some kind of data structure to define, in a given year, which days should count and which should not. IF this is something we really need to track, that solves for different jurisdictions, conventions, etc., BUT then you need to actually know the calendar days you're excluding when you pick something like "business days" for example. 

There could also be some kind of unstructured, manual flag like the unspecified triggers where we have a way to flag that something can't be processed automatically. Needs more discussion.
